lib/Driver/UnixToolChains.cpp

```diff
@@ -325,6 +325,11 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   Arguments.push_back(
       context.Args.MakeArgString(context.Output.getPrimaryOutputFilename()));
 
+  // Run clang++ in verbose mode if "-v" is set
```

Comment:
Oops, sorry. This violates the comment above! Please put the `-v` before the `-o` (in both implementations).

Reply:
Done.